docs/UI/Parsley-Sectioning.md

```diff
@@ -0,0 +1,38 @@
+# Parsley Sectioning
+
+## Overview
+Parsley sectioning is toggleable from the "Log Viewing" tab of the "Parsley Settings" menu and is available to task logs only. Toggling the feature will render your task log into expandable sections split up by function and command.
+
+## Feature details
+
+### Feature components
+At the top of the log viewer, two buttons are available to "Open all sections" and "Close all sections". Clicking these buttons will expand and collapse every function and command section in the log file.
+
+![Toggle All Sections](../images/toggle_all_sections.png)
+
+There are also toggles available on function rows, "Open Subsections" and "Close Subsections", to expand and collapse all commands that belong to the function.
+
+![Toggle Subsections](../images/toggle_subsections.png)
+
+Clicking on the caret will also open and close a single command or function section.
+
+![Toggle Single Section](../images/section_toggle.png)
```

Comment:
I think you can omit this image (since it looks kinda odd just having the image of the caret) and just say
```suggestion
The caret on the left-hand side can also be used to open or close a single command or function section.
```
which the reader can infer from the picture directly above